biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/Traverser.java

```diff
@@ -206,8 +207,9 @@ private void parseResource(Archive archive, String parent) throws Exception {
 			}
 			addInformationCapability(rb, archive.toString(), parent);
 		} catch (Exception e) {
+			Throwable theException = Exceptions.unrollCause(e);
```

Comment:
This will unroll all the way back to the root cause. If you only want to unroll off any InvocactionTargetExceptions, then you want to do:

```java
Throwable theException = Exceptions.unrollCause(e, InvocationTargetException.class);
```

See for example: 

https://github.com/bndtools/bnd/blob/a6d93a4af6b9c388044f632887f682b30d5e8fe6/biz.aQute.bndlib/src/aQute/bnd/osgi/Processor.java#L419-L420

Reply:
Maybe one day I will manage to create PR that does not need at least 3 iterations :-)

Reply:
I think there is a rumour that Chuck Norris could once bypass BJ with only 2 iterations ...